Classes/EventSourcedRepository.php

```diff
@@ -69,27 +81,21 @@ public function save(AggregateRootInterface $aggregate)
     {
         try {
             $stream = $this->eventStore
-                ->get($aggregate->getAggregateIdentifier());
+                ->get($this->generateStreamName($aggregate->getAggregateIdentifier()));
         } catch (EventStreamNotFoundException $e) {
-            $stream = new EventStream(
-                $aggregate->getAggregateIdentifier(),
-                TypeHandling::getTypeForValue($aggregate),
-                []
-            );
+            $stream = new EventStream();
         } finally {
```

Comment:
the `finally` clause doesn't make sense here, it is even a problem: the`catch` clause will only catch `EventStreamNotFoundException` but if some other exception occurs (for example some PDO exception), `$stream` will be undefined.